cypress/e2e/page/annotation-pagetests.cy.js

```diff
@@ -161,192 +166,184 @@ describe("tests on annotation page ui with programmatic state loading and store
 
                     // B. Enter annotated values for each unique value in the 'Diagnosis'-categorized column
                     // NOTE: This value of '3' should be pulled from the interface via the number of unique diagnosis values returned
-                    const uniqueValueCount = 3;
-                    for ( let index = 0; index < uniqueValueCount; index++ ) {
-
-                        cy.get("[data-cy='vocab-term-Diagnosis-" + parseInt(index) + "']")
-                            .click()
-                            .type("Annotated value " +  parseInt(index));
-                    }
-
-                    // C. Click on the 'Save Annotation' button
-                    cy.get("[data-cy='save-button-Diagnosis']")
-                        .click();
+                    cy.get("[data-cy='categoricalSelector_0']").click().type("Depressive{enter}");
+                    cy.get("[data-cy='categoricalSelector_1']").click().type("Parkins{enter}");
+                    cy.get("[data-cy='categoricalSelector_2']").click().type("Other{enter}");
 
                     // 5. Assert annotation nav and next page button are enabled
                     cy.assertNextPageAccess("download", true);
                 }
             });
 
-            it("Annotate single tool group with single assessment tool column; no missing values", () => {
+            // it("Annotate single tool group with single assessment tool column; no missing values", () => {
```

Comment:
As mentioned before: I am against keeping code around in a commented out way. We have it in the git log, so we can always retrieve it when we need it. There can be exceptions, but they should be rare and only used if you get to whatever is commented out next - even then there are better ways to park the code (other branch and pick, git stash if local only, ...). But at a minimum you should provide a TODO comment with a clear explanation why this is commented out and when it can be uncommented.

In this specific case, I think a better solution would be to use `it.skip(...)` to skip the test, and add the TODO comment to explain why it is skipped - but I still think not having the test until we need it is probably better.

Reply:
@jarmoza I'll leave this decision up to you.

Reply:
Oh no. Let's delete this for sure! Thanks for catching that @surchs!
I'll make the change now.